content/v2.0/reference/release-notes/influxdb-cloud.md

```diff
@@ -10,9 +10,34 @@ aliases:
   - /cloud/about/release-notes
 ---
 
+InfluxDB Cloud updates occur continually. Find a compilation of monthly updates below.
+
+## 2020-09-02
+
+## Features
+
+### Pricing updates and Azure region
+
+- Update [pricing vectors](/influxdb/v2.0/account-management/pricing-plans/#pricing-vect) to determine pricing by the total data out and query count.
+
+- Add [Microsoft Azure support](/influxdb/v2.0/reference/urls/#microsoft-azure) for the `eastus` and `westeurope` regions. Each region has a unique InfluxDB Cloud URL and API endpoint.
+
+### Flux updates
+
+ - Add time-weighted average [`timeWeightedAvg()` function](/influxdb/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/timeweightedavg/).
+ - Update [`integral()` function](/influxdb/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/integral/) with linear interpolation.
+ - Add Flux query statistics to better understand query performance.
+ - Add [`tasks.lastSuccess()` function](/influxdb/v2.0/reference/flux/stdlib/influxdb-tasks/lastsuccess/) to retrieve the time of the last successful run of an InfluxDB task.
+ - Add the [`array.from` function](influxdb/v2.0/reference/flux/stdlib/experimental/array/from/) to build ad hoc tables in a Flux script.
+
+### Bug fixes
+
+ - Resolve issues in checks and notifications.
```

Comment:
@gunnaraasen @bpalani do we want to expand on what these issues were?

Reply:
I'm fine with this level of detail for this update. We'll work on more detail summaries of the specific issues resolved in the future as we improve how we summarize the changes for these monthly updates.